docs/docs/README.md

```diff
@@ -3,25 +3,127 @@ prev: false
 next: false
 ---
 
-# What is Pipedream?
+# Overview
 
-Pipedream is an integration platform for developers to build and run [workflows](/workflows/) that integrate apps and data.
+## What is Pipedream?
 
-- Develop any workflow, based on any [trigger](workflows/steps/triggers/).
-- Workflows are code, which you can run for [free](/pricing/).
-- Workflows run serverlessly: you manage no infrastructure of your own.
+Pipedream is a production scale serverless platform to connect APIs, remarkably fast. 
```

Comment:
I think "production-scale" should be hyphenated

Reply:
done